be/src/storage/lake/metacache.cpp

```diff
@@ -206,6 +206,24 @@ std::shared_ptr<const DelVector> Metacache::lookup_delvec(std::string_view key)
     }
 }
 
+bool Metacache::lookup_aggregation_partition(std::string_view key) {
+    auto handle = _cache->lookup(CacheKey(key));
+    if (handle == nullptr) {
+        g_dv_cache_miss << 1;
```

Comment:
`g_dv_cache_miss` is used for delvec, use a new one?

Reply:
> `g_dv_cache_miss` is used for delvec, use a new one?

done.